packages/smarthr-ui/src/components/Textarea/Textarea.tsx

```diff
@@ -135,15 +137,22 @@ export const Textarea = forwardRef<HTMLTextAreaElement, Props & ElementProps>(
       }
     }, [autoFocus])
 
-    const onKeyUp = useMemo(
-      () =>
-        maxLetters
-          ? (event: React.KeyboardEvent<HTMLTextAreaElement>) => {
-              setCount(getStringLength(event.currentTarget.value))
-            }
-          : undefined,
-      [maxLetters],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    const debouncedUpdateCount = useCallback(
+      debounce((value) => {
+        setCount(getStringLength(value))
+      }, 200),
+      [],
     )
+
+    const handleOnChange = useCallback(
```

Comment:
```suggestion
    const handleOnChange = useCallback(
```
nits: Changeイベントのハンドラなので、変数は `handleChange` がよさそうです！

Reply:
@oti ありがとうございます！修正します！

Reply:
@oti requestChangeでレビューいただいていたので、aprvいただけますと 🙇